components/EventsSectionSummary.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div
+        class="relative bg-grey-000 pt-8 -mt-8 pb-16 px-4 sm:px-6 lg:pt-12 lg:px-8"
+    >
+        <a id="community-events" href="/#community-events" class="anchor"></a>
```

Comment:
I don't think we need this anchor here?

Reply:
Wait no, I was just confused 😂 we do need this anchor, nevermind

Reply:
All good. You were right. We don't need this anchor tag in EventsSection and ResourcesSection components.